proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -34,6 +36,12 @@ service Msg {
 
 // MsgCreateValidator defines a SDK message for creating a new validator.
 message MsgCreateValidator {
+  // NOTE(fdymylja): this is a particular case in which
+  // if validator_address == delegator_address then only one
+  // is expected to sign, otherwise both are.
+  option (cosmos.msg.v1.signer) = "delegator_address";
+  option (cosmos.msg.v1.signer) = "validator_address";
```

Comment:
What do you think of `option (cosmos.msg.v1.signer) = "delegator_address,validator_address";` which would be similar to e.g. the orm indices syntax

Reply:
Considering the instances in which we have multiple signers, I'd keep it this way, also because the [decision](https://github.com/cosmos/cosmos-sdk/pull/10977#discussion_r787833302) to rename the field `signers` to `signer` was mainly driven by the most common use case.

But no strong opinion otherwise, let me know.